CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 idf_component_register(
     SRCS "src/bitmap.c" "src/clip.c" "src/fontx.c" "src/hagl.c"  "src/hsl.c"  "src/rgb565.c"  "src/rgb888.c"  "src/tjpgd.c"
     INCLUDE_DIRS "./include"
-    REQUIRES hagl_hal
+    REQUIRES hagl_esp_mipi
```

Comment:
Instead of changing this you should checkout the submodule into folder named hagl_hal

```
$ cd components
$ git submodule add git@github.com:tuupola/hagl_esp_mipi.git hagl_hal 
```

This is explained in the [README](https://github.com/tuupola/hagl_esp_mipi#usage). The reason is that only the ESP HAL is named `hagl_esp_mipi`. HAGL can be used with other platforms too. I have not found a better way to handle this with CMake based build systems.